web/src/views/admin/AdminRepos.vue

```diff
@@ -67,3 +63,9 @@ const { doSubmit: repairRepos, isLoading: isRepairingRepos } = useAsyncAction(as
   notifications.notify({ title: i18n.t('admin.settings.repos.repair.success'), type: 'success' });
 });
 </script>
+
+<style scoped>
+.admin-repos {
```

Comment:
The `ListItem` class item is used in the "repo list" and "admin repos" list (in the admin settings). Both have different paddings `p-0` and `p-4` currently. These differences are not directly respected with tailwind as the overrides are not applied in the same way.

The (only) approach that worked was to use scoped CSS classes instead of competing tailwind classes. This was also recommended by AI.
Not sure why it worked in Windi in the first place.

There might be additional cases in which such scoped classes could be of help. Currently, many subelements are a collection of competing tailwind classes merged in from parent objects. It is often unclear which one actually take precedence. AFAIU, with scoped classes, these always take precedence.

Reply:
Would like to look into this part. Using scoped classes as hack to enforce the right order sounds weird to me.